core/solver/multigrid.cpp

```diff
@@ -52,6 +52,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "core/components/fill_array_kernels.hpp"
 #include "core/solver/ir_kernels.hpp"
 #include "core/solver/multigrid_kernels.hpp"
+#include "core/solver/solver_base.hpp"
 
 
 namespace gko {
```

Comment:
kcycle kernels below can be removed everywhere

Reply:
I think the kernel can be used when we add the kcycle back. It still contains the corresponding tests, so I will prefer keeping them for find them easily